CovertInfiltration/Src/CovertInfiltration/Classes/X2InfiltrationMod.uc

```diff
@@ -8,12 +8,37 @@
 class X2InfiltrationMod extends X2DataSet config(Infiltration);
 
 var const name INFILPREFIX;
+var const name ITEMPREFIX;
+var const name CATEGORYPREFIX;
+var const name ABILITYPREFIX;
+var const name CHARACTERPREFIX;
 
 var config array<InfiltrationModifier> InfilModifiers;
 
-static function name GetInfilName(name ItemName)
+static function name GetInfilName(name ElementName, EInfilModifierType ElementType)
 {
-	return name(default.INFILPREFIX $ ItemName);
+	local name TypeName;
+
+	switch (ElementType)
+	{
+		case eIMT_Item:
+			TypeName = default.ITEMPREFIX;
+			break;
+		case eIMT_Category:
+			TypeName = default.CATEGORYPREFIX;
+			break;
+		case eIMT_Ability:
+			TypeName = default.ABILITYPREFIX;
+			break;
+		case eIMT_Character:
+			TypeName = default.CHARACTERPREFIX;
+			break;
+		default:
+			`CI_Log("X2InfiltrationMod failed to grab type for " $ string(ElementName));
+			return '';
+	}
+
+	return name(string(default.INFILPREFIX) $ string(TypeName) $ string(ElementName));

```

Comment:
Do we even need `string(default.INFILPREFIX) $`? It's a separate template manager, there can't be collisions anyway